public/js/searchList.js

```diff
@@ -30,28 +30,31 @@ const searchCardTemplate =
     `;
 
 /*
- * When the page loads, fetch initial organization data and render it
+ * When the page loads, fetches initial organization data and render it
  * in cards on the list.
  */
 window.onload = function() {
-  fetchOrganizations("all").then(organizations => {
-    createOrganizationCards(organizations);
-    createMarkers(organizations);
-  })
+  updateResults();
 }
 
 /*
- * Fetches the organization data with the given filters from the server.
- * @param {string} filterText The selected item category with which to filer results for.
+ * Requerys for organization data and refreshes page data.
```

Comment:
Nit: spelled queries. Also add a second ```*``` on the previous line here to make it a JSDoc.